bot/src/export.rs

```diff
@@ -0,0 +1,654 @@
+use {
+    crate::data_center_info::{DataCenterId, DataCenterInfo},
+    crate::rpc_client_utils::{get_vote_account_info, VoteAccountInfo},
+    crate::{
+        classify_producers, db::*, get_confirmed_blocks, get_self_stake_by_vote_account, BoxResult,
+        Config,
+    },
+    bincode::{deserialize, serialized_size},
+    lazy_static::lazy_static,
+    log::*,
+    postgres::types::Json,
+    postgres::{Client, NoTls, Transaction},
+    regex::Regex,
+    serde::{Deserialize, Serialize},
+    serde_json::{Map, Number, Value},
+    solana_client::rpc_client::RpcClient,
+    solana_foundation_delegation_program_cli::get_participants_with_state,
+    solana_sdk::pubkey::Pubkey,
+    solana_sdk::{clock::Epoch, commitment_config::CommitmentConfig, short_vec},
+    std::collections::HashMap,
+    std::convert::TryFrom,
+    thiserror::private::DisplayAsDisplay,
+};
+
+// cut-and-paste from from solana/programs/config/src/lib.rs
+#[derive(Debug, Default, Deserialize, Serialize)]
+struct ConfigKeys {
+    #[serde(with = "short_vec")]
+    pub keys: Vec<(Pubkey, bool)>,
+}
+
+fn get_config_data(bytes: &[u8]) -> Result<&[u8], bincode::Error> {
+    deserialize::<ConfigKeys>(bytes)
+        .and_then(|keys| serialized_size(&keys))
+        .map(|offset| &bytes[offset as usize..])
+}
+// end cut-and-paste from from solana/programs/config/src/lib.rs
```

Comment:
Add `solana_config_program::{ConfigKeys, get_config_data},` in the `use { ... }` block and this can all go away now